### ISSUE TITLE
Allow multiple instances of Autopsy to run in parallel

### ISSUE THREAD

**Issue by kefir-** [sleuthkit/autopsy#83]
If I try to run a second instance of Autopsy on my computer, I get the warning:

"An instance of the program seems to be already running with your user directory."

I often work on multiple cases in parallel, and even if Autopsy supports opening multiple cases, the occasional hang/crash that would close down or stop everything is a drawback. EnCase supports multiple open cases, but I always open one instance for each case.

**Comment by DrewcifersHub** [comment]
I agree, multiple instances natively would be nice. But you can achieve this by taking advantage of the fact that it runs based on user context by creating multiple analyst users and using "runas /user:DOMAIN\Username *autopsy executable path*.exe" to run mutliple instances. This is ran in powershell for windows, but there are like commands for linux.
